test/composables/useThemeComposition.test.ts

```
@@ -0,0 +1,139 @@
+      h2: 4,
+      h3: 3,
+      h4: 4,
+      h5: 5,
+      h6: 6,
+    })
+    expect(theme.getOperationBadges()).toEqual(['operationId'])
+    expect(theme.getI18nConfig().locale.value).toBe('es')
+
+    theme.reset()
+
+    expect(theme.getJsonViewerDeep()).toBe(Infinity)
```

Biome

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)